### ISSUE TITLE
Refresh button on flex page does not refresh the number of iterations for current session.

### ISSUE THREAD

**Issue by ghost** [AMOSus/amos-ss16-proj6#68]
Differen sessions have different amounts of iterations. When a different session is selected on the flex page the dropdown for the iteration selection does not refresh. The page has to be reloaded completely for that to happen.
<img width="839" alt="bildschirmfoto 2016-06-22 um 10 05 02" src="https://cloud.githubusercontent.com/assets/18442111/16259054/0db23fd4-3861-11e6-8e26-ff55739f11c5.png">
<img width="637" alt="bildschirmfoto 2016-06-22 um 10 04 49" src="https://cloud.githubusercontent.com/assets/18442111/16259055/0dd134fc-3861-11e6-96b0-ef009dad0603.png">

**Comment by ghost** [comment]
This is a thing in every graph page we think @TobiDo

**Comment by deadlmaster** [comment]
Yes we need to fix this, problem is that the "options" above the chart are not refreshed when a new session is selected. So, Carriers, Iterations and Dimensions are from the very first time, the page got loaded
